cmd/data.go

```diff
@@ -49,6 +49,7 @@ type DataCmd struct {
 	dryRun          bool
 	logLevel        string
 	SkipForeignKeys bool
+	verbose         bool
```

Comment:
This is not needed since the `log-level=DEBUG` is going to handle printing logs enabled via verbose.